src/CoreWCF.Primitives/src/CoreWCF/IAuthorizeOperation.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CoreWCF.Description;
+using CoreWCF.Dispatcher;
+
+namespace CoreWCF
+{
+    public interface IAuthorizeOperation //Any good name would be helpful
+    {
```

Comment:
Remove comment. This name is fine. I think this interface should be in the CoreWCF.Description namespace as should the AuthorizeRoleAttribute class as they are involved in annotating and describing the service.